js/script.js

```diff
@@ -118,27 +118,27 @@ function addItem() {
 
 //Clicking on the close button hides the task
 var close = document.getElementsByClassName("close");
-for(i = 0; i < close.length; i++){
-	close[i].onclick = function (){
-		var div = close[i].parentElement;
-		console.log(div);
-		div.remove();
-	}
+for (i = 0; i < close.length; i++) {
+  close[i].onclick = function () {
+    var div = close[i].parentElement;
+    console.log(div);
+    div.remove();
```

Comment:
As you've learned, `remove()` only hides the element, doesn't remove it from the DOM